iib/web/api_v1.py

```diff
@@ -314,23 +314,33 @@ def get_builds():
     return flask.jsonify(response)
 
 
-@api_v1.route('/healthcheck')
-def get_healthcheck():
+@api_v1.route("/status", methods=["GET"])
```

Comment:
Why are we removing `/healthcheck` endpoint?
This change would cause bumping API version. And I thought we do not want to do it if is not necessary. 
Adding things is OK, but deleting or changing might cause troubles.

Reply:
Oh! yeah so, I changed it to `status` when there were functions for retrieving the version of the pkgs as it would be more suitable than healthchek. Now, that version won't be part of this, I would change it back to the healthcheck.